src/kaleidoscope/MatrixAddr.h

```diff
@@ -57,17 +57,16 @@ class MatrixAddr {
   //       ridiculously bad assembler code for each copy construction,
   //       that would bloat the default firmware by 1K of PROGMEM!
   //
-  constexpr MatrixAddr(const ThisType &other) = default;
-  constexpr MatrixAddr(ThisType &&other) = default;
+  constexpr MatrixAddr(const ThisType &other) = default; // NOLINT(runtime/explicit)
+  constexpr MatrixAddr(ThisType &&other) = default; // NOLINT(runtime/explicit)
```

Comment:
For this two, we do _not_ want explicit constructors, because that would make many of the callsites unreadable, or we'd need to have many more variants of the constructor. That's just not worth it, so it's better to ignore cpplint here.